### PR TITLE
[IMP] module to manage locations in RMA

### PR DESCRIPTION
**Pull request by yaniaular**
This modules work to manage the RMA locations

### CODE REVIEW

**Comment by yaniaular** [comment]
We are working on this PR to make it perfect to merge with OCA/rma.
The modules presented here are part of a collection of modules for the operations of RMA improved by Vauxoo. Each module will be presented in separate PRs because the PR #26 was too heavy for review.. Some dependencies needed.

File: crm_rma_advance_location/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    'version': '1.0',
+    'category': 'Generic Modules/CRM & SRM',
+    'depends': ['crm_claim_rma',
+                'crm_rma_location_rma',
+                ],
+    'author': 'Akretion',
+    'license': 'AGPL-3',
+    'website': 'http://www.akretion.com',
+    'description': """
+RMA Claim Advance Location
```
Comment (pedrobaeza) [inline crm_rma_advance_location/__openerp__.py]: Extract to a README.rst file

File: crm_rma_advance_location/stock.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, SUPERUSER_ID
+from openerp.tools.translate import _
+
+
+class stock_warehouse(models.Model):
```
Comment (pedrobaeza) [inline crm_rma_advance_location/stock.py]: CamelCase

File: crm_rma_advance_location/stock.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        'stock.location',
+        'Carrier Loss Location')
+    lot_breakage_loss_id = fields.Many2one(
+        'stock.location',
+        'Breakage Loss Location')
+    lot_refurbish_id = fields.Many2one(
+        'stock.location',
+        'Refurbish Location')
+
+    def init(self, cr):
```
Comment (pedrobaeza) [inline crm_rma_advance_location/stock.py]: Replace this with a post-init hook. See an example here:
1. https://github.com/odoomrp/odoomrp-wip/blob/8.0/quality_control_stock/__openerp__.py#L44
2. https://github.com/odoomrp/odoomrp-wip/blob/8.0/quality_control_stock/__init__.py#L10

File: crm_rma_advance_location/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    ],
+    'data': [
+        'wizards/claim_make_picking_from_picking_view.xml',
+        'wizards/claim_make_picking_view.xml',
+        'views/crm_claim.xml',
+        'views/stock_picking.xml',
+        'views/stock_warehouse.xml',
+    ],
+    'installable': True,
+    'application': True,
```
Comment (osvalr) [inline crm_rma_advance_location/__openerp__.py]: @yaniaular this need to be marked as an application?

Comment (yaniaular) [inline crm_rma_advance_location/__openerp__.py]: @osvalr no, it is not neccessary. Change it.

**Comment by gurneyalex** [comment]
travis red -> needs fixing

**Comment by osvalr** [comment]
@nhomar this depends on https://github.com/OCA/rma/pull/56 to be merged

**Comment by osvalr** [comment]
Hi @nhomar @yaniaular,

Please review this ASAYC

**Comment by yaniaular** [comment]
:+1:

**Comment by nhomar** [comment]
@osvalr 

Can you add a little description and/or video about what this module does please.

It is clear in the redme but a video can illustrate its power better.

Regards.

**Comment by osvalr** [comment]
Hi @nhomar @pedrobaeza,
I've discussed with @yaniaular the possibility to unify functionalities included in this module (`crm_rma_advance_location`) and `crm_rma_stock_location` (it's currently proposed at OCA/rma#37) based on the following reasons:
- This module adds locations that `crm_rma_stock_location` may use, they does not depends to each other.
- `crm_rma_stock_location` doesn't do much if is installed alone, there is no products or claim lines, also if it's installed alone it seems more a stock (or warehouse) module than a RMA module, so having (and sharing) it's dependencies with this one (`crm_rma_advance_location`) would be the complements to really help to keep track RMA forecasted and on-hand product quantities like it's supposed it does.

**Comment by nhomar** [comment]
@osvalr agreed do the merge please.

**Comment by osvalr** [comment]
Hi @nhomar from now on, this PR is not needed anymore, such functionalities proposed here can be located at OCA/rma#37, so this can be closed then.

**Comment by osvalr** [comment]
**`UPDATE:`** OCA/rma#67 includes this proposal

**Pull request closed without merging**